src/main/java/me/karakelley/http/utility/HtmlGenerator.java

```diff
@@ -0,0 +1,15 @@
+package me.karakelley.http.utility;
+
+import me.karakelley.http.PublicDirectory;
+
+import java.io.File;
+
+public class HtmlGenerator {
+  public static String createLink(File fileName) {
+    return "<p><a href=\"/" + PublicDirectory.relativePath(fileName) + "\">" + fileName.getName() + "</a></p>";
```

Comment:
There is some tight coupling between `HtmlGenerator`, `PublicDirectory`, and `StaticFilesController`